### PR TITLE
[stable/rabbitmq-ha] Update version rabbitmq exporter to version 0.29.0

### PR DESCRIPTION
**Pull request by batazor**
Signed-off-by: Victor Login <batazor111@gmail.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/rabbitmq-ha/values.yaml [hunk 1]
```diff
@@ -399,7 +399,7 @@ prometheus:
     env: {}
     image:
       repository: kbudde/rabbitmq-exporter
-      tag: v0.28.0
+      tag: v0.29.0
```
Comment (davidkarlsen) [inline stable/rabbitmq-ha/values.yaml:419]: please update the readme too

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by davidkarlsen** [comment]
@batazor please fix merge conflict

File: stable/rabbitmq-ha/values.yaml (hunk 1 above)
Comment (batazor) [reply stable/rabbitmq-ha/values.yaml:419]: done

**Comment by batazor** [comment]
@davidkarlsen done

**Comment by steven-sheehy** [comment]
Can you bump version again?

**Comment by batazor** [comment]
@steven-sheehy @davidkarlsen done

**Comment by steven-sheehy** [comment]
You bumped the appVersion mistakenly. Please revert and bump the chart version instead.

**Comment by batazor** [comment]
@steven-sheehy  done

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**